userspace/libscap/scap.c

```diff
@@ -121,7 +121,8 @@ scap_t* scap_open_live_int(char *error, int32_t *rc,
 			   void* proc_callback_context,
 			   bool import_users,
 			   const char *bpf_probe,
-			   const char **suppressed_comms)
+			   const char **suppressed_comms,
+			   bool *syscalls_of_interest)
```

Comment:
> 1) should have length == PPM_SC_MAX 

Indeed the length is SYSCALL_TABLE_SIZE

Btw do you mean something like:
```
typedef struct {
	bool syscalls[SYSCALL_TABLE_SIZE];
} interesting_syscalls_set;
```
?

Reply:
yeah (and with some comments in the declaration of the struct). I think it would help devs that will use this interface.

Reply:
I am not sure if we should overturn `syscalls_of_interest` meaning, ie: calling it `uninteresting_syscalls`, making it explicit that the default behavior is to instrument low level driver to trace them all.
What do you think?

Reply:
Since the values in the array are bool true/false, the struct isn't really a positive or negative filter. I think a generic name like "syscalls_filter" or even "syscalls_of_interest", like you currently have, both sound okay to me.

Reply:
> yeah (and with some comments in the declaration of the struct). I think it would help devs that will use this interface.

Should be ok now :) 
Thanks!